modules/test/playwright/tests/batch-planner/import.spec.ts

```diff
@@ -359,17 +361,13 @@ test('can import CSV file with an unexisting field', async ({
 	]);
 
 	await apiHelpers.objectAdmin.deleteObjectDefinition(response.id);
-	await apiHelpers.featureFlag.updateFeatureFlag('COMMERCE-8087', false);
-	await apiHelpers.featureFlag.updateFeatureFlag('LPS-173135', false);
```

Comment:
Here test is turning 2 FF off however in the beginning it's enabling only one. I don't see any dependency across them in portal.properties, so to me this is a bit of a mystery

Reply:
Oh yes, I forgot to mention it: the `LPS-173135` comes from and old poshi test that was migrated to playwright. However that FF doesn't exist any more so I've simply ignored it.

My guess is that it was a dependency FF that needed to be disabled after the test because there was no support for dependent FFs in poshi (but take it as a hypothesis).